docs/Troubleshooting.md

```diff
@@ -111,19 +111,28 @@ reasons. To expedite resolution of stack deployment issues, it's helpful to
 have the following data and output from the following [AWS CLI](https://aws.amazon.com/cli/) 
 commands when contacting support@quiltdata.io.
 
-1. Installed Quilt stack template metadata: This is located in the
-CloudFormation Quilt stack Outputs tab with the Key `TemplateBuildMetadata`
-and is a JSON object generated by the Quilt build system.
-
-![TemplateBuildMetadata Screenshot](imgs/cloudformation-stack-metadata.png)
-
-2. Quilt stack drift:
-
-    `$ aws cloudformation describe-stack-resource-drifts --stack-name QUILT_STACK_NAME`
+1. Quilt stack outputs:
+    <!--pytest.mark.skip-->
+    ```sh
+    STACK_NAME="YOUR_QUILT_STACK"
+    aws cloudformation describe-stacks \
+        --stack-name "$STACK_NAME" \
+        --query 'Stacks[].Outputs'
+    ```
 
-3. Recent events for the Quilt stack:
+2. Quilt stack drift status:
+    <!--pytest.mark.skip-->
+    ```sh
+    aws cloudformation describe-stack-resource-drifts \
```

Comment:
I think prior to this strict detection had to be started with `detect-stack-drift` and use should wait till it finished.

Reply:
Good point; fixed